Lib/email/policy.py

```diff
@@ -140,9 +140,11 @@ def header_store_parse(self, name, value):
         """
         if hasattr(value, 'name') and value.name.lower() == name.lower():
             return (name, value)
-        if isinstance(value, str) and len(value.splitlines())>1:
+        
+        if isinstance(value, str) and len(linesep_splitter.split(value))>1:
             # XXX this error message isn't quite right when we use splitlines
             # (see issue 22233), but I'm not sure what should happen here.
+            # issue 22233 resolved this by using the linesep_splitter.
```

Comment:
Instead of adding a comment, the XXX comment can just be removed.  Anyone who needs the history should be able to find it from the commit messages (which is where the information from this comment should go).

Don't consider this a blocking comment, please, as my ability to respond to comments is currently somewhat limited.